citesphere/src/main/java/edu/asu/diging/citesphere/core/authority/impl/ViafAuthorityImporter.java

```diff
@@ -110,6 +113,7 @@ public IImportedAuthority retrieveAuthorityData(String uri)
                 String name = iterator.next().getText();
                 ImportedAuthority authority = new ImportedAuthority();
                 authority.setName(name);
+                authority.setImporterId(viafImporter);
```

Comment:
this should be the ID (line 42). The id should then be mapped to the name in the properties file and then name shown on the website.